htdocs/compta/paiement_charge.php

```diff
@@ -51,7 +51,7 @@
  * Actions
  */
 
-if ($action == 'add_payment' || ($action == 'confirm_paiement' && $confirm == 'yes')) {
+if ($action == 'add_payment') {
 	$error = 0;
```

Comment:
It seems to be dead code.
However in develop there is this line :
`if (($action == 'add_payment' || ($action == 'confirm_paiement' && $confirm == 'yes')) && $user->hasRight('tax', 'charges', 'creer')) {`

@eldy ?